src/simpleTrianglePatterns/simpleTrianglePattern2.java

```diff
@@ -8,8 +8,10 @@ public static void main(String[] args){
         System.out.print("Enter a number:");
         try{
             int number = userInputScanner.nextInt();
-        } catch (Exception e) {
+        } catch (WrongUserInputException wuie) {
```

Comment:
Own exception class not bad but this code will crash at a known case. also @cleanCode - meaningful names? e-wuie-rte???